book/src/pong-tutorial/pong-tutorial-02.md

```diff
@@ -269,47 +259,6 @@ compiles. Update the `on_start` method to the following:
 # }
 ```
 
-Let's run our blank screen game!
```

Comment:
The example doesn't contain this bit and `register` doesn't seem to be a `World` method anymore, so I assume this part is no longer applicable.